### DISCUSSION TITLE
Export statement may only appear at top level in file

### DISCUSSION THREAD

**Discussion by sendsnow** [mobxjs/mobx#3265] [Q&A]
hi,everybody,Here's a question for help
In the react native project,run the 'npm i' command with such an error message:

error node_modules/mobx/lib/mobx.module.js: Export statement may only appear at top level in file node_modules/mobx/lib/mobx.module.js at 4420:2. Run CLI with --verbose flag for more details.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! idc_mobile_console@1.7.6 postinstall: `npm run b2`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the idc_mobile_console@1.7.6 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/sx/.tnpm/_logs/2022-01-12T03_36_33_238Z-debug.log
[tnpm] subprocess:/usr/local/lib/node_modules/tnpm/lib/npm_child_process.js exit code: 1, signal: null
[tnpm] mainprocess exit code: 1

What can I do to solve this problem？

**Comment by mweststrate** [comment]
Please provide a minimal reproduction, not much to go on otherwise

**Reply by mrbluesky888** [reply]
Can you help me see how to solve it

**Reply by iChenLei** [reply]
> [tnpm] subprocess:/usr/local/lib/node_modules/tnpm/lib/npm_child_process.js exit code: 1, signal: null
[tnpm] mainprocess exit code: 1

tnpm ? Are you `Alibaba` employee ? I think you should report your issue to `tnpm` maintainers and not mobxjs team. @helloyyt

**Reply by mrbluesky888** [reply]
你好为啥这么说？

**Reply by yrjwcharm** [reply]
@helloyyt  请问你解决了么

**Reply by yrjwcharm** [reply]
我感觉就是mobx的问题太坑了吧

**Comment by mrbluesky888** [comment]
使用火狐浏览器



发自我的iPhone


------------------ Original ------------------
From: yrjwcharm ***@***.***&gt;
Date: Sat,May 21,2022 9:14 PM
To: mobxjs/mobx ***@***.***&gt;
Cc: 遥远的梦、 ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [mobxjs/mobx] Export statement may only appear at top level infile (Discussion #3265)





 
@helloyyt  请问你解决了么

**Comment by tsiresymila1** [comment]
Same problem for me
